src/components/track_player/JamStation.tsx

```diff
@@ -21,29 +19,14 @@ interface JamStationProps {
     timeSections: List<TimeSection>;
     onTrackListChanged: (trackList: TrackList) => void;
     onRefresh: PlainFn;
-    collapsedButtonClassName?: string;
+    collapsedButtonSx?: SystemStyleObject<Theme>;
 }
 
 interface TrackEditDialogState {
     open: boolean;
     randomID: string;
 }
 
-const onTopTheme = (theme: Theme): Theme => {
```

Comment:
See changes in WithHoverMenu.tsx - moving the tooltip menu lower instead of lifting the player higher.